src/main/java/misc/DataProvider.java

```diff
@@ -101,4 +102,12 @@ public static Pharmacy predefinedPharmacy() {
         return new Pharmacy("PharmacyRx", predefinedAddresses()[0], "(123)-342-4344",
             "pharm@pharmaEmail.com");
     }
+
+    public static Cart predefinedCartItems() {

```

Comment:
```suggestion
    public static Cart predefinedCartWithItems() {

```

CartItems sounds like it returns a List of Items, not a Cart

Reply:
I understand your point. I changed and renamed the method to return carts array to be more consistent with my other DataProvider methods.